scripts/publish.sh

```diff
@@ -11,10 +11,9 @@ if [[ "$YARN_MAJOR" -ge "3" ]]; then
   PACK_CMD="yarn pack --out /tmp/%s-%v.tgz"
   # install is handled by yarn berry pack/publish
   INSTALL_CMD=""
-  LOGIN_CMD=""
```

Comment:
waiting to hear back from @legobeat regarding this change. if it's needed I'll add it back and add a shellcheck disable and a note

Reply:
so yeah, totally safe to take this out